pymesync.py

```diff
@@ -133,6 +134,18 @@ def create_time(self, parameter_dict):
         ``parameter_dict`` is a python dictionary containing the time
         information to send to TimeSync.
         """
+        if (parameter_dict['duration'] < 0):
```

Comment:
This can be simplified to

```
if parameter_dict['duration'] < 0:
    return [{self.error: "time object: duration cannot be negative"}]
```

So you don't have to do a method call for a negative duration. You can also remove the check in `__duration_to_seconds` below